### ISSUE TITLE
Discovery and decision-making: Ubuntu Pro with Extended Security Maintenance

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#463]
During set-up for the current iteration, @alecpm made the following suggestion for both [isaw4](https://github.com/isawnyu/isaw.web/issues/460#issuecomment-1467170369) and [isaw1](https://github.com/isawnyu/pleiades-gazetteer/issues/491#issuecomment-1468590635): 

> Consider subscribing to the (free for non-commercial use up to 5 servers) [Ubuntu Pro with Extended Security Maintenance](https://discourse.ubuntu.com/t/ubuntu-pro-faq/34042), which provides additional package security updates. ISAW currently uses a few packages (including python 2.7, fail2ban, and python-pip) which have additional security updates available via the ESM channel.

This ticket has been created to guide discovery and decision-making around this question. If we choose to go this route, new tickets will be created, one for each relevant server.

Tasks:

- [ ] @paregorios to read up on the subscription, via the link above and then add further discovery tasks to this ticket

**Comment by paregorios** [comment]
I note we may not qualify for "free". Here's the detail:

> this service free of charge to personal users who might have their own, or family, or small business needs which we are glad to support as part of our mission and social impact. There are discount programs for specific use-cases, such as research, education, and academia.

Links provided above do not seem to lead to any pricing information about "research, education, and academia", so I'll broaden the search.

**Comment by alecpm** [comment]
It's also possible that an institution like NYU already has some sort of IT/campus wide subscription that you could piggyback on.

**Comment by paregorios** [comment]
Yes, I've just written to the big NYU-wide IT teams list asking if anybody's taken the plunge yet or talked to Canonical about it. The central cloud services operation is an all-RHEL operation, but there could well be one of the schools that's doing Ubuntu.
